packages/sit-onyx/src/components/OnyxSelectInput/OnyxSelectInput.vue

```diff
@@ -125,7 +125,7 @@ const blockTyping = (event: KeyboardEvent) => {
     <OnyxFormElement v-bind="props" :error-messages="errorMessages">
       <template #default="{ id }">
         <!-- eslint-disable-next-line vuejs-accessibility/click-events-have-key-events, vuejs-accessibility/no-static-element-interactions -- Disabled rules are needed here, because of the click event. Otherwise clicking on the padding or gap won't open the select -->
-        <div class="onyx-select-input__wrapper" @click="emit('click')">
+        <div class="onyx-select-input__wrapper" @click="emit('inputClick')">
```

Comment:
We should be able to remove this click handler here without issues, this will also allow us to remove the "eslint" exception above

```suggestion
        <div class="onyx-select-input__wrapper">
```

Reply:
This is creating side effects with opening the select component.